### PR TITLE
React capstone project - Covid-19 Metrics

### PR DESCRIPTION
**Pull request by NayanKaran**
Hi,

I have implemented the project requirements given in the below link:

https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to Microverse
+
```
Comment (keroloussamy) [inline README.md]: - [x] Kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+## Built With
+
+- HTML, CSS, Javascript
+- Webpack, Stylelint, ESlint, React JS, Redux, Router, Jest, Github Actions
+
+## Live Demo
+
+[https://covid19-matrics-nayankaran.netlify.app/](https://covid19-matrics-nayankaran.netlify.app/)
+
```
Comment (keroloussamy) [inline README.md:12]: - [x] Kindly add a link for a recorded video presenting the features of the project.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @NayanKaran, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (NayanKaran) [reply README.md]: Thank you for your suggestion. I have credited the original designer.

File: README.md (hunk 2 above)
Comment (NayanKaran) [reply README.md:12]: Thank you for your suggestion. I added the presentation video to the readme file.

File: README.md (hunk 1 above)
Comment (mateo951) [reply README.md]: Addressed ✔

File: README.md (hunk 2 above)
Comment (mateo951) [reply README.md:12]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi @NayanKaran, 

Well done making the necessary changes requested by the previous reviewer 🚀
In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

### Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**